src/features/colinks/CoLinksContext.tsx

```diff
@@ -54,19 +54,6 @@ const CoLinksProvider: React.FC<CoLinksProviderProps> = ({ children }) => {
 
   const { data } = useCoLinksNavQuery();
 
-  useEffect(() => {
```

Comment:
I think w/o this the wizard will not act nicely when you are on a different chain, would need to test to see how it behaves